packages/data-api/package.json

```diff
@@ -12,12 +12,10 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "build": "rm -rf dist && run-p \"build:* {@}\" --",
-    "build:src": "babel src --out-dir dist --source-maps --ignore \"src/__tests__/**\" --config-file \"../../babel.config.json\"",
-    "build:types": "tsc",
+    "build": "tsc -p tsconfig-build.json",
```

Comment:
We are using a separate `tsconfig` file when building. This is because when linting/compiling we want to include the `__tests__` directory, but we want to exclude them from the build